dGame/Entity.h

```diff
@@ -299,6 +300,11 @@ class Entity {
 	// Scale will only be communicated to the client when the construction packet is sent
 	void SetScale(const float scale) { m_Scale = scale; };
 
+	/**
+	 * @brief The observable for player entity position updates.
+	 */
+	static Observable<Entity*, const PositionUpdate&> OnPlayerPositionUpdate;
```

Comment:
Consider moving static variables into an unnamed namespace in the cpp file (prevents lazy initialization)

Reply:
Then it wouldn't be accessible from the outside of that cpp file, which is something desired here. Alternative is having a getter for these, would that be preferable?

Reply:
do you have examples of how this would be accessed?  hard to see the full picture here without how you would use it

Reply:
the reason for asking about this is to prevent any future issue where you push stuff into this container, and then if the server crashes, all those observers are removed before the server does its processing of data during the crash.  if that is a non-concern then feel free to dismiss, though i try to verify that if a variable is static, it truly is 100% OK to be fully destructed before the crash handler does its work.